packages/orchestrator-ui-components/src/pages/metadata/WfoWorkflowsPage.tsx

```diff
@@ -46,6 +57,8 @@ export type WorkflowListItem = Pick<
 
 export const WfoWorkflowsPage = () => {
     const t = useTranslations('metadata.workflows');
+    const tError = useTranslations('errors');
```

Comment:
This seems like a to much repitition between files. Can't some of this be abstracted into a useExport hook? Togheter with the parts where the data is being retrieved?